apps/emqx_dashboard/src/emqx_dashboard_api.erl

```diff
@@ -211,8 +213,13 @@ user(delete, #{bindings := #{username := Username}}) ->
         true -> {400, #{code => <<"CONNOT_DELETE_ADMIN">>,
                         message => <<"Cannot delete admin">>}};
         false ->
-            _ = emqx_dashboard_admin:remove_user(Username),
-            {200}
+            case emqx_dashboard_admin:remove_user(Username) of
+                {error, not_found} ->
```

Comment:
are you sure it returns `{error, not_found}` ?
```
(aa@zmd)4> mnesia:transaction(fun() -> mnesia:abort(foo) end).
{aborted,foo}
```

Reply:
Yes, it will return `{error, not_found}`. At `emqx_dashboard_admin.erl` line 148, changed error type.